dbt/adapters/duckdb/credentials.py

```diff
@@ -155,6 +155,15 @@ def is_motherduck(self):
     def _is_motherduck(scheme: str) -> bool:
         return scheme in {"md", "motherduck"}
 
+    @property
+    def motherduck_token(self) -> str:
```

Comment:
So I'm a little thrown by this-- I thought the normal place folks put their MD token was in the `path` setting as opposed to by using the MD plugin configuration. There may be more places we want to look for the token besides the plugin config?

Reply:
I moved it out of this file entirely :)